README.md

```diff
@@ -1,5 +1,8 @@
 # ocular
 
-Ocular is a tool primarily designed for building documentation websites for github-based javascript frameworks, built using the gatsbyjs documentation generation system.
+Ocular is a monorepo for development tools primarily designed for building github-based javascript frameworks.
```

Comment:
With this major change to move towards using gatsby themes,  it worth considering moving gatsby-theme-ocular out of the ocular monorepo?  I don't see lots of overlapping code or scripts between dev tools and ocular-gatsby, and the move to themes seems to further diverge the components.  Maybe too much work at this point, but worth a discussion.

Reply:
>  Is it worth considering moving gatsby-theme-ocular out of the ocular monorepo? 

Yes, though if anything should be moved out to another repo, it is probably the `dev-tools` module, since the `ocular` name (and this repo) was originally created for the first (non-gatsby) version of our documentation generator.

An argument for keeping them here is that each repo comes with a certain amount of overhead. While code sharing is limited, these modules both follows a theme of being developer automation type tools, and one could imagine it being a home for additional automation tools in the future.

Also, a big push in the recent v8 time frame has been to consolidate into fewer repos, examples:
- `viewport-mercator-project` went into `@math.gl/web-mercator`
- `seer` went into `@probe.gl/seer`
- `monochrome` into `streetscape.gl` 
- etc.